packages/core/core/src/loadParcelConfig.js

```diff
@@ -3,7 +3,9 @@ import type {
   FilePath,
   ParcelConfigFile,
   ResolvedParcelConfigFile,
```

Comment:
Is `ResolvedConfigFile` necessary anymore? Since `resolveFrom` used to be used to resolve all plugins, but now each PluginNode should have a `resolveFrom`. I think `filePath` is only used in `ConfigLoader` where it gets set to the `resolvedPath`. It could just be returned in the object returned from processConfig along with `extendedFiles`.

Reply:
`ResolvedParcelConfigFile` is still used for loading the config and starting the loading chain?
PluginNode's are an internal representation of the config, `ResolvedParcelConfigFile` is still used for initialising Parcel through the API